CHANGELOG.md

```diff
@@ -2,10 +2,6 @@
 
 ## 5.1.1
 
-- BUGFIX: Fix issues with Swift templates ([bug 1749494](https://bugzilla.mozilla.org/show_bug.cgi?id=1749494))
```

Comment:
Instead of removing something from a release that was already cut, we should add a new changelog entry explaining that 5.1.1 was broken for Swift and explaining what was reverted.

Reply:
Hehe, oops I just clicked the revert button. Nice catch!